content/product-engineering/engineering/cloud/saas/index.md

```diff
@@ -88,6 +88,14 @@ The Cloud SaaS team works alongside several other teams within the Cloud organiz
 
 ## News and updates
 
+### Weekly team updates
+
+Each Friday, we send weekly team updates to the rest of the Cloud org to keep the Cloud senior leadership and our partner teams in the loop about Cloud SaaS Team progress and our ups and downs. You can find all the weekly updates in this [google doc](https://docs.google.com/document/d/1a_36dJ-ZR4LU_3bDaIWAZldZJ6O3zaSi8NhGfT0fsdw/edit#heading=h.yggics8n0cyt).
+
+### Newsletter
+
+We are committed to sending a monthly newsletter to the entire Product and Engineering org summarising our progress towards current goals, challenges, opportunities, learning, and important team updates. You can find all the previous entries in the links below:
```

Comment:
Do we have a doc / process for curating the newsletter? If so, we should leave information that we need to update links here every month, or we can just link to https://groups.google.com/a/sourcegraph.com/g/engineering-team-status/search?q=Cloud%20SaaS%20monthly%20update.

Reply:
So far we wrote one and it was me doing it. I'm planning to do another one this week. Long term I'd love to share this responsibility with the whole team. I agree that having this process documented would be great. I will make a TODO not for myself to update our handbook with the process and do this contribution as separate PR